core/src/main/java/com/linecorp/armeria/common/AggregatedHttpResponse.java

```diff
@@ -40,6 +41,9 @@
      * Creates a new HTTP response.
      *
      * @param statusCode the HTTP status code
+     *
+     * @throws IllegalArgumentException if the specified {@code statusCode} is
+     *                                  {@link HttpStatus#isInformational()}.
```

Comment:
How about using `{@linkplain}`?

```suggestion
     * @throws IllegalArgumentException if the specified {@code statusCode} is
     *                                  {@linkplain HttpStatus#isInformational() informational}.
```